apps/api/src/domain/games/memory-match/queries/memory-match.controller.ts

```diff
@@ -19,11 +23,28 @@ import { MemoryMatchService } from '../services/memory-match.service';
 export class MemoryMatchController {
     constructor(private readonly memoryMatchService: MemoryMatchService) {}
 
+    // TODO this should be a MemoryMatchRoundViewModel
+    @ApiResponse({
```

Comment:
Once you have that view model, you'll need to update `MemoryMatchService.buildView` as follows:
```ts
private buildView(memoryMatchRound): MemoryMatchRoundViewModel{
     return new MemoryMatchRoundViewModel(
        memoryMatchRound,
     {
         buildUrl: (id: AggregateId) =>{
                    return `${this.configService.get('BASE_URL')}:${this.configService.get(
            'NODE_PORT'
        )}/${this.configService.get('GLOBAL_PREFIX')}/resources/mediaItems/download/${mediaItemId}`;
        }
     }
    )
}
```